### PR TITLE
fix pulsar admin cli tool

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

- Fix the token examples of CLI tools
- Add the example of pulsar-client for connecting to a cluster either through Oauth2
- Add the example of pulsar-admin for connecting to a cluster either through Oauth2
- Add the example of pulsar-perf for connecting to a cluster either through Oauth2

### CODE REVIEW

File: cloud/pulsar-client/README.md [hunk 1]
```diff
@@ -4,9 +4,11 @@ The `pulsar-client` and `pulsar-perf` are CLI tool written in Java language for
 
 # Prerequisites
 
-- Pulsar broker 2.7.0-742fc5c9b+
+> Pulsar client is required to be newer than 2.6.1 which will include the OAuth2 authentication plugin.
 
-> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
+- Pulsar broker 2.7.0-742fc5c9b+
+- Get the `SERVICE_URL` of your StreamNative Cloud Pulsar cluster: [How to get service URL](#How to get service URL)
```
Comment (jiazhai) [inline cloud/pulsar-client/README.md:10]: where is `#How to get service URL` and `#How to get token options` tracked?

Comment (wolfstudy) [reply cloud/pulsar-client/README.md:10]: fix, PTAL again

File: cloud/pulsar-admin/README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar
 
 # Usage
 
+## Token
```
Comment (jiazhai) [inline cloud/pulsar-admin/README.md:15]: Please help change all the "Token/Oauth2" into  "Token/Oauth2 authentication plugin" in these 2 files?

File: cloud/pulsar-admin/README.md [hunk 3]
```diff
@@ -22,3 +24,29 @@ The `pulsar-admin` supports to connect to Pulsar cluster through Token, as shown
...
+```
+
+## OAuth2
+
+The `pulsar-admin` supports to connect to Pulsar cluster through OAuth2, as shown below:
+
+```shell script
+bin/pulsar-admin --admin-url https://cloud.streamnative.dev:443 \
+  --auth-plugin org.apache.pulsar.client.impl.auth.oauth2.AuthenticationOAuth2 \
+  --auth-params '{"privateKey":"file:///path/to/key/file.txt",
```
Comment (jiazhai) [inline cloud/pulsar-admin/README.md:41]: file.txt is a json format right?  how about change it into file.json?  If changing this, please also change similar places.

File: cloud/pulsar-admin/README.md [hunk 4]
```diff
@@ -4,19 +4,49 @@ The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar
 
 # Prerequisites
 
-- Pulsar broker 2.7.0-742fc5c9b+
+> Pulsar client is required to be newer than 2.6.1 which will include the OAuth2 authentication plugin.
 
-> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
+- Pulsar broker 2.7.0-742fc5c9b+
+- Get the `WEB_SERVICE_URL` of your StreamNative Cloud Pulsar cluster: [How to get service URL](../README.md#How to get service URL)
+- Get the `AUTH_PARAMS` of your StreamNative Cloud Pulsar cluster: [How to get token options](../README.md#How to get token options)
```
Comment (jiazhai) [inline cloud/pulsar-admin/README.md:11]: Is the format of the link right? 
https://github.com/streamnative/pulsar-examples/blob/c83e2818bdb1a51cf72558fbc52d54fc2537cc55/cloud/pulsar-admin/README.md
image

**Comment by wolfstudy** [comment]
@jiazhai PTAL again, thanks

**Pull request merged**